### DISCUSSION TITLE
T:Slim X2 Integration

### DISCUSSION THREAD

**Discussion by Miquest** [NightscoutFoundation/xDrip#2276] [General]
Hey @jwoglom and @tolot27,
how is the T:Slim integration going?  What are the problems and does it seem possible at all so far?
I need an update, I'm so excited ;)

**Comment by jwoglom** [comment]
@thorinEichenschild007 for updates check the Nightscout/WeAreNotWaiting discord in the #tslim-x2-dev channel. https://discord.gg/KyZGdhyt44

**Reply by Miquest** [reply]
Okay, I'll join this server as soon as possible
